### PR TITLE
Fix initial bottom panel size

### PR DESCRIPTION
**Pull request by mortal94**
Issue:
The initial bottom panel size on small screens was too big

## What I did
Changed the initial value of the bottom panel size to be dynamic to the screen.

### CODE REVIEW

File: lib/ui/src/components/layout/container.js [hunk 1]
```diff
@@ -282,7 +282,7 @@ class Layout extends Component {
       resizerPanel:
         resizerPanel ||
         (options.panelPosition === 'bottom'
-          ? { x: 0, y: bounds.height - 400 }
+          ? { x: 0, y: bounds.height * 0.6 }
```
Comment (ndelangen) [inline lib/ui/src/components/layout/container.js]: It would be nice to round this to a whole number, to not get inconsistencies

**Review by ndelangen — CHANGES_REQUESTED** [review]

**Comment by ndelangen** [comment]
Thanks @mortal94 this is great, one small remark, then good to merge IMHO

**Comment by mortal94** [comment]
> Thanks @mortal94 this is great, one small remark, then good to merge IMHO

Thanks @ndelangen ! i've pushed the requested changes. 
Will this fix be released in v5.1-beta.2?

**Review by ndelangen — APPROVED** [review]

**Comment by shilman** [comment]
@mortal94 yes this will be released in beta.2 after it gets merged

**Pull request merged**